pkg/kubelet/apis/podresources/v1alpha1/api.proto

```diff
@@ -0,0 +1,43 @@
+// To regenerate api.pb.go run hack/update-generated-pod-resources.sh
+syntax = 'proto3';
+
+package v1alpha1;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+option (gogoproto.goproto_stringer_all) = false;
+option (gogoproto.stringer_all) =  true;
+option (gogoproto.goproto_getters_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.goproto_unrecognized_all) = false;
+
+
+// PodResourcesLister is a service provided by the kubelet that provides information about the
+// node resources consumed by pods and containers on the node
+service PodResourcesLister {
+    rpc List(ListPodResourcesRequest) returns (ListPodResourcesResponse) {}
+}
+ // ListPodResourcesRequest is the request made to the PodResourcesLister service
+message ListPodResourcesRequest {}
+ // ListPodResourcesResponse is the response returned by List function
+message ListPodResourcesResponse {
+    repeated PodResources pod_resources = 1;
+}
+ // PodResources contains information about the node resources assigned to a pod
```

Comment:
Is there a space before your comment? In that case, nit: Remove the space :)

Reply:
There was, nice catch.